packages/web-react/src/components/ButtonLink/ButtonLink.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import classNames from 'classnames';
+import { WithChildren } from '../../types/main';
+import { compose } from '../../utils/compose';
+import { applyColor, applyClassNamePrefix } from '../../utils/classname';
+import { useClassNamePrefix } from '../../hooks/useClassNamePrefix';
+
+type Color = 'primary' | 'secondary' | 'tertiary';
+
+export interface ButtonLinkProps extends WithChildren {
+  /**
+   * Emotion color variant.
+   */
+  color: Color;
+  /**
+   * HTML Anchor `href` attribute.
+   */
+  href: string;
+  /**
+   * HTML Anchor `target` attribute.
+   */
+  target?: string;
+  /**
+   * Block alignment modification
+   */
+  block: boolean;
+  /**
+   * Disabling button
+   */
+  disabled: boolean;
+  /**
+   * Aria label
+   */
+  ariaLabel?: string;
+  /**
+   * Component CSS class
+   */
+  componentClassName: string;
+  className?: string;
+}
+
+// `${componentClassName}--${color}`;
+const getButtonColorClassname = (componentClassName: string, color: Color): string =>
+  compose(applyColor<Color>(color))(componentClassName);
+
+export const ButtonLink = ({
+  color,
+  href,
+  target,
+  block,
+  ariaLabel,
+  disabled,
+  className,
+  componentClassName,
+  ...restAttributes
+}: ButtonLinkProps): JSX.Element => {
+  const classNamePrefix = useClassNamePrefix();
+  const mainClassName = applyClassNamePrefix(classNamePrefix)(componentClassName);
+
+  return (
+    <a
+      {...restAttributes}
+      href={href}
+      target={target}
+      className={classNames(
+        mainClassName,
+        getButtonColorClassname(mainClassName, color),
+        {
+          'Button--block': block,
+        },
+        {
+          'Button--disabled': disabled,
```

Comment:
This class is missing prefix application.

Reply:
incorporated